server/routes/user.py

```diff
@@ -85,7 +86,18 @@ def index():
     return "Data Commons </br> <a href='/user/auth/login'><button>Login</button></a>"
 
 
-@bp.route('/upload/import', methods=['POST'])
+@bp.route('/imports')
+@login_is_required
+def get_imports() -> Dict[str, Dict]:
+    user_id = libutil.hash_id(session['oauth_user_id'])
+    user = user_api.get_user(user_id)
+    imports: Dict[str, Dict] = {}
+    for im in user.reference.collection(user_api.IMPORT_COLLECTION).stream():
+        imports[im.id] = im.to_dict()
+    return imports
+
+
+@bp.route('/import/upload', methods=['POST'])
```

Comment:
Should we start an API file for this route?

Reply:
This is a user action and now with full URL /user/import/upload. Maybe can leave it here for now and as more endpoints added, we can decide if have a new file for user import actions?

Reply:
sounds good, but please add a TODO so we add one when the next endpoint is added.

Reply:
Done